Press release, about page, and site formatting

Hexel opened the pull request:
- Added Connor's canvassing event at MLK Jr Station to list of upcoming events in press release
- Refined formatting of press release content
- Changed press release author from "DATA Team" to "DATA Press Release" and gave it a new author avatar
- Edits to About page
- Edits to home page layout
- Edits to Post page layout

The layout changes were mainly to address how ginormous and spaced out everything is when viewed on a full-size monitor. After all, we're all about density, right? Compare preview links to see before and after.

LEQUI000 commented:
Images are significantly better on mobile and don't take up as much space. 

Also love the new About Us section

CurtisGreen commented:
@Hexel Thanks for making a PR!

I think the previous design had a bit too much margin but this is a bit too small, maybe add a little margin above the hero post
![image](https://github.com/user-attachments/assets/554c0421-e0e2-43b3-bc94-56f3286239a6)

I like that you made the hero post smaller and switched the text to the right side

I think I like previous design's "More Stories" section a bit better because it makes more of a distinction between the hero post and the other stories. It might look a little strange right now since there's only 1 story down there but once there's 2+ I think it looks nice

I like the changes to the "About Us" page, do you think we should remove the period at the end of the mission statement?

democat3457 commented:
I agree with Curtis on the previous design of the "More Stories" section, and honestly, there should probably be a post carousel for when we do get more posts published - this would reduce the vertical space that the section takes up while still showing sufficient posts.

For the about page, I feel like the last paragraph should be written completely in past tense or be moved into some other non-history section, as the history section should not contain ongoing matters.

LEQUI000 commented:
I just made a minor update to the PR with some changes (see #16) that @Schobbish made to add a relevant banner to the site based on a post—I also removed the phone number that previously mentioned a null phone number.

CurtisGreen commented on src/app/globals.css:
```diff
@@ -1,3 +1,20 @@
+}
+
+h1, h2, h3, h4, h5, h6 {
+    text-wrap: pretty;
```
I need to setup formatting rules but currently the project is using a 2-space format I believe

jadannanwosu commented:
I like how stripped down it is. The improved compactness is definitely a plus. But along with a +1 to the original "More Stories" section, I think this home page feels too much like a personal blog. Not a UI/UX expert by any means, but I'd like some sort of visual element that ties together our message as a buffer between the top menu and the posts. So you can tell immediately this is an Organization with Authority!

LEQUI000 commented:
> I agree with Curtis on the previous design of the "More Stories" section, and honestly, there should probably be a post carousel for when we do get more posts published - this would reduce the vertical space that the section takes up while still showing sufficient posts.

Is it not possible to make it show the more recent 3-5 articles then a show more tab would "load more articles as available" or "load a new page of an archive of all articles"

CurtisGreen commented on src/app/globals.css:
```diff
@@ -1,3 +1,20 @@
+
+    a {
+        text-decoration: underline;
+        text-decoration-style: dotted;
```
I usually see dotted used for tooltips rather than links, so I think we should use the standard underline

democat3457 replied:
+1 on this, I also believed the dotted underlines were for tooltips for a split second before I hovered over the link

CurtisGreen commented on _posts/data-announces-formation.md:
```diff
@@ -55,8 +61,8 @@ Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)
 Phone: (214) 555-1234  
```
I know this has been talked about but just a reminder to remove this if we don't have a number to use

LEQUI000 replied:
Phone number removed.

democat3457 replied:
Why did you revert the line break I added?

LEQUI000 replied:
That might have been unintentional. Apologies

democat3457 commented:
Also @Hexel you can put two line breaks after the first line in a commit to add a commit description... no need for the title to be so long lol
<img width="860" alt="image" src="https://github.com/user-attachments/assets/bf5565ae-de5b-4506-b967-5ca8a6db743a">

jadannanwosu commented on src/app/page.tsx:
```diff
@@ -9,9 +9,6 @@ export default function Index() {
   return (
     <>
-      <h1 className="my-16 md:mb-12 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
-        We are the ridership.
```
It doesn't have to be as huge as it was, but maybe this tagline can come back as the simple visual element I was talking about? Something like "We are the ridership.   Est. 2024"

Schobbish replied:
I liked that too

Hexel replied:
I'm not sure I like "We are the ridership" because that feels more boastful than I like. Cuz I'm imagining if, in the near-term, we pull 20 people out to a council hearing, that would be a successful turnout for us this early on! But if we say "we are the ridership" that anchors us to the 200K ridership number, which makes 20 feel small.

The full slogan I was thinking of with the name DATA was "_We are the data you're looking for_" or the shorter version "_We are the data_." It's a play on words because there are several possible interpretations and they're all correct.
- We are the DATA = We are the Dallas Area Transit Alliance
- We are DATA = We are Dallas Area Transit Alliance
- We are the data you're looking for = We are the evidence that real people use DART and it's not just empty buses.
- We are data = We are what caused the trip counter to increment.
- We are the DATA you're looking for = We are the transit alliance in the Dallas area that you sought out after recently being orange pilled and started Googling for local transit advocacy groups.
- We are data you're looking for = Are you a reporter looking to interview people without cars / people with disabilities / people who depend on transit / students who use transit / any transit-related demographic to inform your journalism? We are your one-stop shop for finding real people to talk to.

I could go on. "We are the data" is akin to "You are not in traffic; you are the traffic."

Hexel commented:
> I agree with Curtis on the previous design of the "More Stories" section, and honestly, there should probably be a post carousel for when we do get more posts published

Gonna have to be that guy who breaks the old news: [carousels are bad. It's conclusive.](https://www.gschroeder.com/blog/2022/5/1/carousels-are-bad-its-conclusive). 

> this would reduce the vertical space that the section takes up while still showing sufficient posts.

How often do we expect the website will be updated with new content? I assume it won't take the place of r/dart for casual posts. The frequency would determine what is considered a "sufficient" number of posts to display at once. News sites show many stories above the fold because they have multiple new stories daily. In contrast, our return visitors likely won't see more than two new stories each time they visit.

> For the about page, I feel like the last paragraph should be written completely in past tense or be moved into some other non-history section, as the history section should not contain ongoing matters.

Agreed. If situations change, the About Us page will become outdated. They can read the petition details directly, so we don't need to repeat them on the About page. I'll remove it.

> I like how stripped down it is. The improved compactness is definitely a plus. But along with a +1 to the original "More Stories" section, I think this home page feels too much like a personal blog. Not a UI/UX expert by any means, but I'd like some sort of visual element that ties together our message as a buffer between the top menu and the posts. So you can tell immediately this is an Organization with Authority!

Noted! I really like how [www.ridersalliance.org](https://www.ridersalliance.org/) is structured. I see our website serving two functions:
- Electronic brochure: For people who want to learn more about our organization.
- Corporate blog: For announcements, press releases, and editorials that reflect our collective opinion (not personal opinions, which should be saved for Reddit).

Schobbish commented on _posts/data-announces-formation.md:
```diff
@@ -1,23 +1,22 @@
 ogImage:
-  url: "assets/blog/data-announces-formation/logo-transparent.png"
+  url: "/assets/blog/data-announces-formation/data-meeting-july-20.webp"
+banner: "[Read our latest press release announcing our formation.]($slug)"
```
The format of this is still changing so it will need to be updated when I'm done with my feature. @LEQUI000

LEQUI000 replied:
Got it. Will be on the lookout.

Hexel commented:
- Removed parts of About Us page that describe events in progress
- Converted indents from 4 to 2 spaces

- Changed inline link styling from dotted to solid underline

- Removed null phone number (merged from @LEQUI000)

- Reverted layout of More Stories section on home page

- Increased spacing above and below hero post on home page

CurtisGreen reviewed (approved):
I think we'll need to trim our cover images to all be the same size for better alignment, but that can be done in a separate PR. 

Nice work 👍

democat3457 commented on src/app/about/page.tsx:
```diff
@@ -18,7 +18,7 @@ export default function Index() {
         <p>The idea for DATA emerged during a meeting on June 15, 2024, at Cole Park in Uptown Dallas, where approximately 15 people gathered to explore the possibility of forming a group. A second meeting on July 10, 2024, at the Metropolitan Building in Downtown Dallas saw over 50 attendees, during which interim co-chairs Tyler Wright and Connor Hulla were elected, and the organization's priori ...
 
-        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition has garnered over 1,160 signatures and counting. As stated in the petit ...
+        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition garnered over 1,160 signatures and counting. The petition called upon t ...
```
```suggestion
        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition garnered over 1,160 signatures in two weeks. The petition called upon the DART Board of Directors to reject any proposals to reduce the sales tax contributions from the thirteen member cities.</p>
```
